samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -153,7 +153,12 @@ public void processAction(ActionEvent ae) throws AbortProcessingException, FinFo
                         taskService.completeUserTaskByReference(reference, userIds);
                         
             if (adata.getForGrade()) {
-                Event event = eventTrackingService.newEvent(SamigoConstants.EVENT_ASSESSMENT_AUTO_GRADED, adata.getPublishedAssessmentTitle(), null, true, NotificationService.NOTI_OPTIONAL, SamigoLRSStatements.getStatementForGradedAssessment(adata, publishedAssessment));
+                StringBuilder logString = new StringBuilder();
+                logString.append("siteId=");
```

Comment:
the siteId would already be part of the event via 3rd param, seems unnecessary to add again here.

Reply:
True, it's in the context.
![image](https://github.com/sakaiproject/sakai/assets/8653754/a3e55029-3a63-4c52-a80d-c832348ca3b6)